projects/commudle-admin/src/app/feature-modules/events/components/collaborating-communities/collaborating-communities.component.html

```diff
@@ -19,10 +19,35 @@
       </div>
     </div>
     <div *ngIf="event.editable">
-      <input #autoInput (input)="onChange()" [nbAutocomplete]="auto" [value]="selectedCommunity" nbInput
-             placeholder="Enter Name..." type="text"/>
-      <nb-autocomplete #auto (selectedChange)="onSelectionChange($event)" [handleDisplayFn]="autocompleteDisplay">
-        <nb-option *ngFor="let option of communities" [value]="option"> {{ option.name }} </nb-option>
+      <label for="nb-input">
+        <p>
+          <nb-icon icon="info-circle" pack="fas"></nb-icon>
+          Type 3 or more characters to get the results
+        </p>
+      </label>
+
+      <input #autoInput
+      nbInput
+      id="nb-input"
+      type="text"
+      [value]="selectedCommunity"
+      (input)="onChange()"
+      (keyup)="checkTyping()"
+      placeholder="Enter Name..."
+      [nbAutocomplete]="auto" />
+
+      <nb-autocomplete #auto (selectedChange)="onSelectionChange($event)">
+
+        <nb-option *ngFor="let community of communities" [value]="community">
+          <div class="logo">
+            <img src="{{community.logo_path}}" alt="{{ community.name }}" title="{{ community.name }}">
+          </div>
+          {{ community.name }}
+        </nb-option>
+        <nb-option disabled *ngIf="communities.length===0 && typing">
```

Comment:
Can we change this condition so that 'No results' shows only after the actual search takes place (like search happens only after 3 characters are typed right, so let's show no results after that)?